src/views/DrefApplicationForm/Overview/CopyFieldReportSection/index.tsx

```diff
@@ -160,32 +157,29 @@ function CopyFieldReportSection(props: Props) {
                 }
             }
 
-            onValueSet({
-                ...value,
-                go_field_report_date,
-                disaster_type,
-                event_description,
-                un_or_other_actor,
-                national_society_contact_name,
-                national_society_contact_email,
-                national_society_contact_phone_number,
-                national_society_contact_title,
-                ifrc_emergency_name,
-                ifrc_emergency_email,
-                ifrc_emergency_phone_number,
-                ifrc_emergency_title,
-                media_contact_name,
-                media_contact_email,
-                media_contact_phone_number,
-                media_contact_title,
-                field_report: fieldReportResponse.id,
-                country,
-                district,
-                num_affected,
-                partner_national_society,
-                ifrc,
-                icrc,
-            });
+            setFieldValue(go_field_report_date, 'go_field_report_date');
+            setFieldValue(disaster_type, 'disaster_type');
+            setFieldValue(event_description, 'event_description');
+            setFieldValue(un_or_other_actor, 'un_or_other_actor');
+            setFieldValue(national_society_contact_name, 'national_society_contact_name');
+            setFieldValue(national_society_contact_email, 'national_society_contact_email');
+            setFieldValue(national_society_contact_phone_number, 'national_society_contact_phone_number');
+            setFieldValue(national_society_contact_title, 'national_society_contact_title');
+            setFieldValue(ifrc_emergency_name, 'ifrc_emergency_name');
+            setFieldValue(ifrc_emergency_email, 'ifrc_emergency_email');
+            setFieldValue(ifrc_emergency_phone_number, 'ifrc_emergency_phone_number');
+            setFieldValue(ifrc_emergency_title, 'ifrc_emergency_title');
+            setFieldValue(media_contact_name, 'media_contact_name');
+            setFieldValue(media_contact_email, 'media_contact_email');
+            setFieldValue(media_contact_phone_number, 'media_contact_phone_number');
+            setFieldValue(media_contact_title, 'media_contact_title');
+            setFieldValue(fieldReportResponse.id, 'field_report');
+            setFieldValue(country, 'country');
+            setFieldValue(district, 'district');
+            setFieldValue(num_affected, 'num_affected');
+            setFieldValue(partner_national_society, 'partner_national_society');
+            setFieldValue(ifrc, 'ifrc');
+            setFieldValue(icrc, 'icrc');
```

Comment:
need to batch

Reply:
This is already batched, so it should be fine!